tests/integration/tests.mk

```diff
@@ -79,15 +79,7 @@ test.integration.local: | $(JUNIT_REPORT)
 test.integration.local.presubmit: | $(JUNIT_REPORT)
 	mkdir -p $(dir $(JUNIT_UNIT_TEST_XML))
 	set -o pipefail; \
-	$(GO) test -p 1 ${T} ${TEST_PACKAGES} --istio.test.env native --istio.test.select +presubmit \
-	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
-
-# Unstable/flaky/new integration tests targeting local environment.
-.PHONY: test.integration.local.unstable
```

Comment:
I'm assuming that the unstable target was never used?

Reply:
correct. I was planning to add them as post submit gates. We can either do that, or run everything together in post-submit.